src/features/shell-sync/main/compute-unix-shell-environment.global-override-for-injectable.ts

```diff
@@ -6,6 +6,7 @@
 import { getGlobalOverride } from "../../../common/test-utils/get-global-override";
 import computeUnixShellEnvironmentInjectable from "./compute-unix-shell-environment.injectable";
 
-export default getGlobalOverride(computeUnixShellEnvironmentInjectable, () => async () => {
-  throw new Error("Tried to get unix shell env without override");
-});
+export default getGlobalOverride(computeUnixShellEnvironmentInjectable, () => async () => ({
+  callWasSuccessful: true,
+  response: process.env,
```

Comment:
This is introducing side effect to override. In global override the response should be always same.